### ISSUE TITLE
Ocean culling incorrect with Register Height Input

### ISSUE THREAD

**Issue by daleeidd** [wave-harmonic/crest#865]
**Describe the bug**
Ocean culling becomes more incorrect the further the height is changed by _register height input_.

**Screenshots / video**
<img width="1110" alt="123131" src="https://user-images.githubusercontent.com/5249806/123713312-f8d60880-d828-11eb-9771-a0609aea1676.png">
This is a 100x100 quad with "set base water height using geometry" just under 99m above sea level.

**Versions**
All RPs and latest master.

**To Reproduce**
1. Create a quad 100x100
2. Add _Register Height Input_ with appropriate material (set or base both exhibit the issue)
3. Set quad height to 99m above sea level
4. Incorrect culling should occur when close enough

**Platform**
 - Editor
 - Mac and Windows

Haven't tested standalone yet.

**Additional context**
The issue is probably incorrect displacement reporting. Also, any _register height input_ further than 99m above sea level appears to be ignored.

**Comment by daleeidd** [comment]
Turns out it wasn't reporting the height because I didn't set either reporting options. @huwb Should report renderer bounds be on by default?

Also the 100m issue because of:
```
_worldToCameraMatrix[lodIdx] = CalculateWorldToCameraMatrixRHS(_renderData[lodIdx]._posSnapped + Vector3.up * 100f, Quaternion.AngleAxis(90f, Vector3.right));
```

So basically anything over 100 above sea level is getting clipped. I don't know if this is something you plan on solving elsewhere like with water bodies?

**Comment by daleeidd** [comment]
Fixed in 4.13.
